lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -0,0 +1,105 @@
+# encoding: UTF-8
+require 'kafo_parsers/doc_parser'
+
+module KafoParsers
+  class PuppetStringsModuleParser
+    # You can call this method to get all supported information from a given manifest
+    #
+    # @param [ String ] manifest file path to parse
+    # @return [ Hash ] hash containing values, validations, documentation, types, groups and conditions
+    def self.parse(file)
+      content = new(file)
+      docs    = content.docs
+
+      # data_type must be called before other validations
+      data = {
+        :object_type => content.data_type,
+        :values      => content.values,
+        :validations => content.validations
+      }
+      data[:parameters] = data[:values].keys
+      data.merge!(docs)
+      data
+    end
+
+    def initialize(file)
+      @file = file
+      raise KafoParsers::ModuleName, "File not found #{file}, check your answer file" unless File.exists?(file)
+
+      command = "puppet strings #{file} --emit-json-stdout"
```

Comment:
we could add a list of paths to test so even system puppet would be supported, I guess that would be `/usr/bin/puppet` on deb and rpm systems right?